Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: test
+
+test:
+	./samba-integration-centos-ci-tests.sh ghp ghprbPullId=''
```

Comment:
you pass these are parameters to the script, but you want to (un)set them as environment variables instead

Reply:
THanks. updated. Now it works.